docs/details/shards/General/Count.md

```diff
@@ -0,0 +1,5 @@
+`Count` parses the value passed to it in the `:Name` parameter, and returns the count of characters, elements, or key-value pairs depending on whether the data type passed to it was a string, a sequence, or a table. 
+
+If this shard is applied to a number it returns zero as the count. Parameters `:Key` and `:Global` are not applicable to `Count`. 
```

Comment:
Similarly to `Clear`,  `:Global` and `:Key` should work here. Can you add samples for those cases?

Reply:
`:Global`

Same behviour as for `(Clear)` above:

`(:Global true)` doesn't give a count
```clj
[1 2 3] >= .seq
(Count .seq :Global true) (Log)    ;; => 0
```

`(:Global false)` and not passing `:Global)` at all gives a valid count
```clj
[1 2 3] >= .seq
(Count .seq :Global false) (Log)    ;; => 3
```
```clj
[1 2 3] >= .seq
(Count .seq) (Log)                  ;; => 3
```

Reply:
Usage of `:Global` documented in commit [#0ecc5a77767aad041cd24a60afb1f15f83bc2a39](https://github.com/fragcolor-xyz/shards/pull/305/commits/0ecc5a77767aad041cd24a60afb1f15f83bc2a39)

Usage of `:Key` shall be documented in issue #335